build.gradle

```diff
@@ -118,7 +118,7 @@ spotless {
 		enforceCheck false
 	}
 	else {
-		String spotlessBranch = "origin/main"
+		String spotlessBranch = "origin/3.4.x"
```

Comment:
note: this will need to be reverted back to `origin/main` on the main branch when forward-merging